src/internal/observable/empty.ts

```diff
@@ -1,5 +1,4 @@
 import { Observable } from '../Observable';
```

Comment:
Nit (maybe completely unimportant, but still): since many other files contain both uppercase and lowercase letters in file name, for consistency reasons, this one could be renamed like this: `empty.ts` -> `EMPTY.ts`. I'm not sure if it makes sense to do it, this is just a reminder. If it makes sense, the same renaming would be needed for all `-spec` and `NEVER` related files.

Reply:
I agree. I think we probably need a pass to do git moves for a lot of files that are named silly things, honestly. But we can probably do that in another PR.